src/themes/vue.styl

```diff
@@ -103,28 +103,85 @@ body
 .markdown-section em
   color #7f8c8d
 
-.markdown-section code
+.markdown-section code,
+.markdown-section pre,
+.markdown-section output::after
+  font-family 'Roboto Mono', Monaco, courier, monospace
+
+.markdown-section code,
+.markdown-section pre
   background-color #f8f8f8
+
+.markdown-section pre,
+.markdown-section output
+  margin 1.2em 0
+  position relative
+
+.markdown-section pre > code,
+.markdown-section output
+  border-radius 2px
+  display block
+
+.markdown-section pre > code,
+.markdown-section output::after
+  -moz-osx-font-smoothing initial
+  -webkit-font-smoothing initial
+
+.markdown-section code
```

Comment:
Are these style changes potentially breaking (website will look different when v4 user updates)?

Reply:
Oh never mind, looks like only refactoring.

Reply:
Non-breaking change because styles did not exist previously.